io.github.kukuruzka165.materialgram.yml

```diff
@@ -5,6 +5,7 @@ sdk: org.freedesktop.Sdk
 command: materialgram
 rename-icon: materialgram
 finish-args:
+  - --filesystem=host
```

Comment:
Why is this required now?

Reply:
Many people have problems with the file drag and drop function and this is a temporary fix (until something changes in upstream telegram)

Reply:
So why would it need host? You are mostly dragging and dropping from home or a few xdg-download, xdg-pictures directories?

Reply:
my bad, will that be enough?
```
- --filesystem=xdg-pictures:ro
- --filesystem=xdg-download:ro
- --filesystem=xdg-desktop:ro
- --filesystem=home:ro
```

Reply:
`home` covers the rest. `- --filesystem=home:ro` is fine, if it works for you.